### PR TITLE
Fix/upgrade finance

### PR DESCRIPTION
**Pull request by tcrosado**
Add missing function to upgrade Ads finance contract without need to redeploy main Ads contract.

### CODE REVIEW

File: test/advertisement.js [hunk 1]
```diff
@@ -455,4 +455,48 @@ contract('Advertisement', function(accounts) {
 		});
```
Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

**Review by GustavoRSSilva — APPROVED** [review]

File: test/advertisement.js (hunk 1 above)
Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

Comment (DioPires) [inline test/advertisement.js:455]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 49 passing
```
#### Test coverage: 
79.26 %

**Pull request merged**